docs/neofs-blockstorage.md

```diff
@@ -103,6 +129,38 @@ supported yet by this command. Please, add a comment to the
 [#3744](https://github.com/nspcc-dev/neo-go/issues/3744) issue if you need this
 functionality.
 
+### NeoFS StateFetcher
+
+NeoFS StateFetcher service enables contract state synchronization for non-archival nodes
+by fetching contract storage key-value pairs from a trusted NeoFS container. 
+It serves as an alternative to P2P-based MPT state synchronization ([P2PStateExchngeExtensions](https://github.com/nspcc-dev/neo-go/blob/f7080f28d7088517de1d624dfdaf247f914486d2/docs/node-configuration.md?plain=1#L421)).
+It allows to download contract state objects from a trusted container in the NeoFS 
+network and persist them to database using standard verification flow. 
```

Comment:
> using standard verification flow. 

What is standard verification flow in terms of storage items? It was relevant for blocks, but we don't have it for storage items.